search-sentiments-map/public/scripts/current-data-script.js

```diff
@@ -117,23 +138,43 @@ function setTimeRange(timeRange) {
     return;
   }
   currentTimeRange = timeRange;
-  updateTrends();
+  getIsWorldLevel() ?
+      updateGlobalTrendsAndDisplayFirst() : updateUsTrendsAndDisplayFirst();
 }
 
 /**
- * Fetches current top trends from back end and displays them on the website.
+ * Fetches current top global trends from the backend, displays them on the 
+ * website, and shows map data for the first trend.
  * @param {boolean=} setNewTrendEnabled Boolean for whether or not to change
  *     the trend that the user is viewing.
  */
-function updateTrends(setNewTrendEnabled = true) {
+function updateGlobalTrendsAndDisplayFirst(setNewTrendEnabled = true) {
```

Comment:
Is it really necessary to say trend again? I just think that is too long. And `updateGlobalTrends` doesn't say that it displays the first trend on the map

Reply:
I am a little confused. It doesn't always display the first trend, therefore we should not say it does in the title.

Reply:
I'm also a little confused, since it changes the map data when there's a new search topic, so it does not always display the first trend? Correct me if I'm wrong

Reply:
But we should say that it will possibly display the first

Reply:
updateGlobalTrendsAndInitiallyDisplayFirstTrend - although repetitive, like Haylee mentioned before method names can be as long as you want

Reply:
see below